chromium/browser_type.go

```diff
@@ -282,7 +282,6 @@ func prepareFlags(lopts *common.LaunchOptions, k6opts *k6lib.Options) (map[strin
 		"disable-background-timer-throttling":                true,
 		"disable-backgrounding-occluded-windows":             true,
 		"disable-breakpad":                                   true,
-		"disable-client-side-phishing-detection":             true,
```

Comment:
Maybe we need an issue to track `safebrowsing-enable-enhanced-protection`?

Reply:
Absolutely. Was on my TODO list :) I will create an issue for it and relate it with #771.